<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
    <addaction name="exitFile"/>
   </widget>
   <widget class="QMenu" name="edite">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="cancel"/>
    <addaction name="separator"/>
    <addaction name="cut"/>
    <addaction name="copy"/>
    <addaction name="paste"/>
    <addaction name="delete_L"/>
    <addaction name="separator"/>
    <addaction name="lookup"/>
    <addaction name="findNext"/>
    <addaction name="replace"/>
    <addaction name="goto_G"/>
    <addaction name="separator"/>
    <addaction name="checkAll"/>
    <addaction name="date"/>
   </widget>
   <widget class="QMenu" name="format">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="wordWrap"/>
    <addaction name="font"/>
    <addaction name="color"/>
   </widget>
   <widget class="QMenu" name="lookOver">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="status"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpView"/>
    <addaction name="separator"/>
    <addaction name="aboutNote"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edite"/>
   <addaction name="format"/>
   <addaction name="lookOver"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="exitFile">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="cancel">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="cut">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="copy">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="paste">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="delete_L">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="lookup">
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
  </action>
  <action name="findNext">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="replace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="goto_G">
   <property name="text">
    <string>转到(&amp;G)...</string>
   </property>
  </action>
  <action name="checkAll">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="date">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="wordWrap">
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="font">
   <property name="text">
    <string>字体(&amp;F)...</string>
   </property>
  </action>
  <action name="status">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="helpView">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="aboutNote">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="color">
   <property name="text">
    <string>颜色(&amp;c)...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
